apis/elbv2/v1alpha1/targetgroupbinding_types.go

```diff
@@ -102,6 +110,10 @@ type TargetGroupBindingSpec struct {
 	// targetGroupARN is the Amazon Resource Name (ARN) for the TargetGroup.
 	TargetGroupARN string `json:"targetGroupARN"`
 
+	// targetType is the TargetType of TargetGroup. If unspecified, it will be automatically inferred.
+	// +optional
+	TargetType *TargetType `json:"targetType,omitempty"`
```

Comment:
What will be the default value? Is it upto the model?

Reply:
it will be automatically decided by a mutatingWebhook using DescribeTargetGroup API call if unset.
When model builders creates this TargetGroupBinding, it will set the TargetType to avoid the extra DescribeTargetGroup call.
When users creates this TargetGroupBinding without specify it, it will be set to the TargetType of TargetGroup by mutatingWebhook.